resources/js/components/Common/Banner.vue

```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
+import { XMarkIcon } from '@heroicons/vue/20/solid'
+defineProps<{text:string}>()
+defineEmits<{ click: [] }>()
```

Comment:
It is for what?

Reply:
To import X icon and make X button functional

Reply:
I mean `defineEmits`

Reply:
I use it to make a function call from banner in a parent